hydra-node/json-schemas/api.yaml

```diff
@@ -787,6 +803,58 @@ components:
       payload:
         $ref: "api.yaml#/components/schemas/TextEnvelopeTransaction"
 
+    ProtocolParameters:
```

Comment:
Should: not define this ourselves

See other comment above.

Reply:
validation works only if we have it defined as a local message type